sdks/python/apache_beam/runners/worker/statesampler_test.py

```diff
@@ -22,21 +22,24 @@
 import time
 import unittest
 
-from nose.plugins.skip import SkipTest
-
+from apache_beam.runners.worker import statesampler
 from apache_beam.utils.counters import CounterFactory
+from apache_beam.utils.counters import CounterName
 
 
 class StateSamplerTest(unittest.TestCase):
 
-  def setUp(self):
+  @classmethod
+  def setUpClass(cls):
     try:
-      # pylint: disable=global-variable-not-assigned
-      global statesampler
-      from apache_beam.runners.worker import statesampler
+      # pylint: disable=unused-variable
+      from apache_beam.runners.worker import statesampler_fast
+      cls.slow_sampler = False
```

Comment:
Actually, instead of duplicating the logic, perhaps just query state_sampler.FAST_SAMPLER directly below.